exporter/redis.go

```diff
@@ -747,6 +756,7 @@ func getKeysFromPatterns(c redis.Conn, keys []dbKeyPair) (expandedKeys []dbKeyPa
 }
 
 func (e *Exporter) scrapeRedisHost(scrapes chan<- scrapeResult, addr string, idx int) error {
+	var config []string
```

Comment:
Same, why move this up?

Reply:
same here

Reply:
I'd prefer to not move config up but keep it in the if as is right now, the shadowing isn't a problem here.